CMakeLists.txt

```diff
@@ -358,6 +355,20 @@ else()
       POSITION_INDEPENDENT_CODE true)
 endif()
 
+
+# Now target jansson is declared, set per-target values
+
+target_compile_definitions(jansson PUBLIC JANSSON_USING_CMAKE)
+target_compile_definitions(jansson PRIVATE HAVE_CONFIG_H)
+
+target_include_directories(jansson PUBLIC
+	$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
+	INTERFACE $<INSTALL_INTERFACE:include>
+)
```

Comment:
Nice, looks good. The formatting seems a bit inconsistent with `PUBLIC` on the line before its argument and `INTERFACE` on the same line as its argument.